medor_custom/models/res_company.py

```diff
@@ -30,7 +30,7 @@ def _compute_company_data(self):
             cooperators = (
                 self.env['res.partner']
                     .sudo()
-                    .search([('member', '=', True)])
+                    .search([('cooperator', '=', True)])
             )
             company.nb_cooperators = len(cooperators)
```

Comment:
Jean-Claude are you aware that you return the all the res_partner concerned by the easy my coop module and not just the effective cooperator ?

Reply:
Mmh, I probably mixed a conflict resolution because I made the mistake before. thx dude.

Reply:
👍